src/engine.c

```diff
@@ -26,7 +26,7 @@ static int start_timeout = 30;
 static int enable_screen_updates = 1;
 static int debug_palette_number = 0;
 
-int engine_init(void) {
+int engine_init(bool headless) {
```

Comment:
This probably should have stdbool.h included?

Reply:
Well, that's possible, but it works for me and for the CI with the current. I can put it there sure.

Reply:
Added stdbool.h